src/github-repo.ts

```diff
@@ -244,7 +245,17 @@ export class GithubRepo {
     }
     // Doing in sequence to not overload GitHub with requests
     for (const asset of assets) {
-      await this._uploadAsset(releaseDetails, asset);
+      try {
+        await this._uploadAsset(releaseDetails, asset);
+      } catch (err) {
+        if ((err as OctokitRequestError)?.status === 500) {
+          // Sometimes GitHub returns ECONNRESET errors, wait a second and retry.
+          await setTimeoutAsync(1000);
+          await this._uploadAsset(releaseDetails, asset);
```

Comment:
Should this fail after a given # of attempts?

Reply:
I was thinking one retry would be enough here. It would make it easier for future changes if we write it that way though, so I'll update it to that way of doing it at least. Do you feel one retry is too little? Maybe 3?

Reply:
Ah sorry, misread the code thinking that it was calling `uploadReleaseAsset`, not `_uploadAsset` – all good. :) No strong preferences between 1 retry and multiple retries, we can always adjust later if we want.

Reply:
There is a `waitFor` method in the same file that we already use for the same thing if you ever want to get back to that